content/en/core/process/workflow.md

```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
+
+### Technical Documentation
+Mainly prepared for a technical audience, it describes the [CHT Core Framework]({{< ref "core" >}}) and its [reference components]({{< ref "apps/reference" >}}). It includes information on technical implementation, design decisions, architecture descriptions, schemas, and source code. 
+
+### How-to Guides and Tutorials
+Mainly prepared for implementing partner capacity building, it includes best practices [quick guides]({{< ref "apps/guides" >}}) and [tutorials]({{< ref "apps/tutorials" >}}) on functional implementation of a feature (or group of features) and system configuration to assist users in completing specific objectives. 
+
+## Issues
+
+All documentation and technical documentation site issues are managed in [Github cht-docs](https://github.com/medic/cht-docs/issues). When creating issues, add the appropriate [labels](https://github.com/medic/cht-docs/labels) to help provide guidance to other community members.
+
+## Prioritization
+
+CHT documentation is treated similar to CHT code: the aim is to improve it as often as possible. Documentation is prioritized in three different ways:
+
+1. Feature education is completed at the time of releasing the related issue in GitHub. At the beginning of each release, an issue(s) is created, it is added to the release project, and a lead is assigned. 
+
+2. Technical documentation is completed at the time of closing the related issue in a GitHub release. 
+
+3. How-to guides and tutorials are prioritized alongside other product work as part of the roadmap planning process.
+
+## Writing
+
+We place a high importance on consistency and usability of CHT documentation. After all, the CHT is an open source software deployed in communities around the globe. The CHT [documentation style guide]({{< ref "docs-style-guide" >}}) will help you to write documentation in the most consistent and useful way.
+
+## Commits
+
+The main branch is master which must be kept stable so as not to impact other documentation writers. To achieve this we do (almost) all documentation changes in a branch and submit a PR for review. This means another documentation writer has signed off on the change before it hits the master branch.
+
+Format your commit messages according to Git standards. First line should be a short title/summary (50 characters or so) with more details in a separate paragraph (if needed).
+
+{{% alert title="Note" %}} Every commit message should be able to complete the following sentence:
+When applied, this commit will: {YOUR COMMIT MESSAGE} {{% /alert %}} 
+
+## Reviewing
+
+The documentation author should assign a reviewer for any documentation changes that are made. We use the following review structure:
+
+* Features education is reviewed by a Product Manager and a Community Manager 
+* Technical documentation is reviewed by an Engineer
+* How-to guides and tutorials are reviewed by a Capacity Building Manager and a Community Manager
```

Comment:
Seems to be missing design system. That said, I think we are better off having this automated by CODEOWNERS mechanism instead of leaving it up to folks to figure out who the reviewer should be. We should instead give instruction to the review as to what they should do, and who should merge. We can work out that language together.